ci-operator/step-registry/gather/cert-rotation/gather-cert-rotation-commands.sh

```diff
@@ -73,7 +73,7 @@ run-on-first-master "
   oc get csr -o yaml > ${ARTIFACT_DIR}/csrs.yaml
   oc get co -o yaml > ${ARTIFACT_DIR}/cos.yaml
   chown -R core:core ${ARTIFACT_DIR}
-  oc --insecure-skip-tls-verify adm must-gather --timeout=15m --dest-dir=${ARTIFACT_DIR}/must-gather
+  oc --insecure-skip-tls-verify adm must-gather --timeout=15m --dest-dir=${ARTIFACT_DIR}/must-gather || true
```

Comment:
Does this mean we skip failed must-gather?

Reply:
Right, but I don't think it works as expected. Testing a fix in https://github.com/openshift/release/pull/47478